ouroboros-consensus-shelley-test/test/Test/Consensus/Shelley/Golden.hs

```diff
@@ -39,13 +39,7 @@ import           Test.Consensus.Shelley.MockCrypto
 
 tests :: TestTree
 tests = testGroup "Golden tests"
-    [ testCase "Block"          test_golden_Block
-    , testCase "Header"         test_golden_Header
-    , testCase "HeaderHash"     test_golden_HeaderHash
-    , testCase "GenTx"          test_golden_GenTx
-    , testCase "GenTxId"        test_golden_GenTxId
-    , testCase "ApplyTxErr"     test_golden_ApplyTxErr
-    , testCase "ConsensusState" test_golden_ConsensusState
```

Comment:
Does `cardano-ledger-specs` actually have golden tests for these?

Reply:
Well hidden, but yes... https://github.com/input-output-hk/cardano-ledger-specs/blob/master/shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Serialization.hs#L1103

Reply:
I see, good. That's where they belong.

(Although we're the ones actually determining what we send across the network, so it's nice to have a mechanism to detect when the format changes in ledger-specs, but we can use hashes for that instead of the full CBOR output. That's something for a different PR, of course. Moreover, in the end, we'll focus the golden tests on `CardanoBlock`, not the individual ledgers, as that's *the* ledger that we'll be running.)